src/System Application/App/Permission Sets/src/PermissionImpl.Codeunit.al

```diff
@@ -97,30 +99,40 @@ codeunit 9864 "Permission Impl."
                             if TenantPermission."Read Permission" <> PermissionOption then begin
                                 TenantPermission."Read Permission" := PermissionOption;
                                 ModifyPermissionLine := true;
+                                Session.LogAuditMessage(StrSubstNo(PermissionUpdatedLbl, RIMDX, TenantPermission."App ID", CopyStr(TenantPermission."Role ID", 1, MaxStrLen(TenantPermission."Role ID")), TenantPermission."Object Type", TenantPermission."Object ID",
+                                    TenantPermission."Read Permission", UserSecurityId()), SecurityOperationResult::Success, AuditCategory::ApplicationManagement, 2, 0);
```

Comment:
Isn't AuditCategory, UserManagement or RoleManagement?

Reply:
This is defined in the AuditMessageOperation parameter (the next to last), but unfortunately it is not Enum, but integer. AuditCategory defines whether this is Authorization, Authentication or AppManagement.